docs/resources/annotation.md

```diff
@@ -0,0 +1,36 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_annotation Resource - terraform-provider-grafana"
+subcategory: ""
+description: |-
+  Official documentation https://grafana.com/docs/grafana/latest/dashboards/annotations/HTTP API https://grafana.com/docs/grafana/latest/http_api/annotations/
+---
+
+# grafana_annotation (Resource)
+
+* [Official documentation](https://grafana.com/docs/grafana/latest/dashboards/annotations/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/annotations/)
+
+## Example Usage
+
+```terraform
+resource "grafana_annotation" "test" {
+  text = "basic text"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `dashboard_id` (Number) The ID of the dashboard on which to create the annotation. Defaults to `false`.
+- `panel_id` (Number) The ID of the dashboard panel on which to create the annotation. Defaults to `false`.
+- `tags` (Set of String) The tags to associate with the annotation.
+- `text` (String) The text to associate with the annotation. Defaults to `false`.
```

Comment:
I think having the time and timeEnd attributes is needed for a MVP. If we're managing annotations with Terraform, they're probably historical annotations, not ones that apply to right now